server/middleware/notificationDismiss.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response } from 'express'
+import { markAsDismissed } from '../services/notificationCookieService'
+
+export default () => async (req: Request, res: Response) => {
```

Comment:
would be good to copy the unit test from DPS too

Reply:
whoops forgot .. done